lib/takeaway.rb

```diff
@@ -0,0 +1,48 @@
+
+class Takeaway
+
+  attr_reader :pizzas, :order
+
+  def initialize
+    @pizzas = [
+      { name: :Margherita, price: 9 },
+      { name: :Napoli, price: 9 },
+      { name: :Marinara, price: 7 },
+      { name: :Pepperoni, price:10 },
+      { name: :Calabrese, price: 10 },
+      { name: :Funghi, price: 8 },
+      { name: :Napoletana, price: 9 }
+    ]
+  end 
+
+  def menu
+  index = 1
+    @pizzas.each do |pizza|
+      puts "#{index}: #{pizza[:name]} (#{pizza[:price]})"
+    end 
+  end 
+
+  def create_order(order = Order.new)
+    @order = order
+  end 
+
+  def add_order(user_order)
```

Comment:
There are some Order methods in the Takeaway class - you can try class extraction to move the Order methods into the Class methods or rename the Order methods to avoid more than one description in a class